salt/orch/etcd-migrate.sls

```diff
@@ -0,0 +1,155 @@
+# To migrate etcd storage version for Kubernestes execute following steps on Admin node:
+# 1. docker exec -ti $(docker ps | grep salt-master | cut -d" " -f1) salt-run state.orchestrate orch.etcd-migrate
+# 2. btrfs property set -ts /.snapshots/1/snapshot ro false
+# 3. mount -o remount,rw /
+# 4. sed -i '/etcd_version/ s/etcd2/etcd3/' /usr/share/salt/kubernetes/pillar/params.sls
```

Comment:
This change will be reverted back to etcd2 the first time we ship an update

Reply:
Ok, thank you Kiall, I wanted to put it under discussion. I will move etcd_version pillar value to database.